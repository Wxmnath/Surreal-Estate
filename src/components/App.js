import "../styles/App.css";
import NavBar from "./NavBar";

function App() {
  return (
    <div className="App">
      <NavBar />
      <header className="App header">Surreal Estate</header>
    </div>
  );
}

export default App;
