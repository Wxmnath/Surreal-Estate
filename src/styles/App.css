@import url("https://fonts.googleapis.com/css2?family=Playfair+Display:ital@1&display=swap");

/* .App {
  display: flex;
  flex-direction: column;
  min-width: 100vh;
} */

body {
  background-color: white;
  font-family: sans-serif;
}
