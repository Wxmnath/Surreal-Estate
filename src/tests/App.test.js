import { render, screen } from "@testing-library/react";
import App from "../components/App";

describe("App", () => {
  xtest("renders title", () => {
    // eslint-disable-next-line react/react-in-jsx-scope
    render(<App />);
    const linkElement = screen.getByText(/Surreal Estate/i);
    expect(linkElement).toBeInTheDocument();
  });
});
