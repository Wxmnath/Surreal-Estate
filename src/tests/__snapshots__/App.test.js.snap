// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders app correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <ul
        class="navbar-list"
      >
        <img
          alt="logo"
          class="logo"
          src="Surreal_Estate_Logo.png"
        />
        <a
          class="navbar-links-item"
          href="/"
        >
          View Properties
        </a>
        <a
          class="navbar-links-item"
          href="/add-property"
        >
          Add a Property
        </a>
      </ul>
    </div>
    <div>
      <h2>
        Properties Page
      </h2>
      <div
        class="sidebar"
      >
        <form
          class="search-by-title"
        >
          <input
            type="text"
            value=""
          />
          <button
            type="submit"
          >
            Search
          </button>
        </form>
        <h5
          class="sidebar-title"
        >
          Filter By City:
        </h5>
        <a
          class="link-item"
          href="/?query={\\"city\\":\\"Manchester\\"}"
        >
          Manchester
        </a>
        <a
          class="link-item"
          href="/?query={\\"city\\":\\"Leeds\\"}"
        >
          Leeds
        </a>
        <a
          class="link-item"
          href="/?query={\\"city\\":\\"Sheffield\\"}"
        >
          Sheffield
        </a>
        <a
          class="link-item"
          href="/?query={\\"city\\":\\"Liverpool\\"}"
        >
          Liverpool
        </a>
        <h5
          class="sidebar-title"
        >
          Sort By Price:
        </h5>
        <a
          class="link-item"
          href="/?sort={\\"price\\":-1}"
        >
          Price Descending
        </a>
        <a
          class="link-item"
          href="/?sort={\\"price\\":1}"
        >
          Price Ascending
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
